homeassistant/components/homekit/type_cameras.py

```diff
@@ -195,7 +195,9 @@ async def _async_get_stream_source(self):
         if stream_source:
             return stream_source
         try:
-            stream_source = await camera.stream_source()
```

Comment:
Can we now also drop usage of `self._camera` ?

Reply:
Right now its checking for `is_on` but since homekit doesn't know about that its not going to change the user experience if I just remove it.